vhdbuilder/packer/components.json

```diff
@@ -142,12 +142,9 @@
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/calico/pod2daemon-flexvol:*",
-      "amd64OnlyVersions": [
-        "v3.8.9.1"
-      ],
+      "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v3.23.3",
-        "v3.24.0"
+        "v3.25.0.2"
```

Comment:
I am not sure if this is what we actually get from our calico operator. Checking....